packages/lib-classifier/src/plugins/drawingTools/experimental/components/TranscriptionLine/components/TranscriptionLineMark/TranscriptionLineMark.js

```diff
@@ -38,10 +38,9 @@ const TranscriptionLineMark = forwardRef((props, ref) => {
       fill={color}
       ref={ref}
       stroke={color}
-      strokeWidth={2}
```

Comment:
[Stroke width](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-width) can’t be set for groups, so I assume this line did nothing.